### ISSUE TITLE
Crash Reporting

### ISSUE THREAD

**Issue by Dachmian** [GetDutchie/datadog_flutter#92]
Hello! 

Thank you for the great package, its working wonderfully for us.

I have a question about crash reporting. This is supported by the native packages for [Android](https://www.datadoghq.com/blog/debug-android-crashes/) and [iOS](https://www.datadoghq.com/blog/ios-crash-reporting-datadog/)

Any plans of supporting this functionality for Flutter? 

Cheers

**Comment by tshedor** [comment]
Hey @Dachmian this is supported through RUM. To handle the exceptions Flutter reports, I'd recommend running your app within a zone - see [Automatically capture errors](https://github.com/GetDutchie/datadog_flutter#real-user-monitoring) here.

**Comment by Dachmian** [comment]
Hi @tshedor,
Can you please clarify if your comment is related to Flutter errors or app crashes in general?

We are using RUM and initalizing it as you suggested inside a zone but we are still having crashes which are visible in Xcode/TestFlight but are not visible in Datadog RUM view.

If you see the links posted above for iOS and Android it seems its possible to configure the nativr SDK to capture all crashes.

**Comment by tshedor** [comment]
Hey @Dachmian for Android, `crashReportsEnabled` is [enabled so long as you're passing a RUM ID](https://github.com/GetDutchie/datadog_flutter/blob/main/android/src/main/kotlin/com/greenbits/datadog_flutter/DatadogFlutterPlugin.kt#L67).

For iOS, it looks like this was updated extremely recently (like ~10 days ago recently). I've added this to a PR. Can you try it out?

```yaml
datadog_flutter:
  git:
    url: git@github.com:GetDutchie/datadog_flutter.git
    ref: add-ios-reporting
```

**Comment by Dachmian** [comment]
@tshedor 
Thank you for the quick turnaround!
We are testing it right now and will get back to you

**Comment by tshedor** [comment]
@Dachmian any news on this?

**Comment by Dachmian** [comment]
Hi @tshedor 
We implemented based on your suggestion and have tested and it works for both iOS and Android. Thanks a lot!

Can this be merged into a new release?

**Comment by tshedor** [comment]
Released in 1.7.3

**Comment by tshedor** [comment]
@Dachmian this is rereleased in `2.0.0-beta.1`
